src/pages/Main/Main.jsx

```diff
@@ -1,7 +1,85 @@
 import React from 'react';
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import CardList from './CardList/CardList';
+import CardListNew from './CardList/CardListNew';
+import MainCarousel from './MainCarousel/MainCarousel';
+import MAIN_DATA from './CardList/MainData.jsx';
 
 const Main = () => {
-  return <article>Main</article>;
+  const [productList, setProductList] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/productData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setProductList(data);
+      });
+  }, []);
+
+  return (
+    <MainWrapper>
+      <MainCarousel />
+      <MainContent>
+        <Suggestion>
```

Comment:
아래 리뷰에 따르면 이렇게 되겠네요!

```suggestion
        <div>
```